spec/hashTable_test.js

```diff
@@ -0,0 +1,91 @@
+import chai, { expect } from 'chai'
+import chaiChange from 'chai-change'
+import HashTable from '../src/HashTable'
+
+chai.use(chaiChange)
+
+describe('HashTable', () => {
+  'use strict'
+
+  it('exists', () => {
+    expect(HashTable).to.be.a('function')
+  })
+
+  describe('put()', () => {
+    it('adds a key-value pair to the hash table.', () => {
+      const hash = new HashTable()
+      console.log(HashTable);
+      hash.put('name', 'philliplorenzo')
+      hash.put('42.0', 'renzo')
+      expect(hash.contains('42.0')).to.be.true
+    })
+  })
+
+  describe('get()', () => {
+    it('returns the data associated with key.', () => {
+      const hash = new HashTable()
+      hash.put('name', 'batman')
```

Comment:
The test need not be this verbose - we can insert one thing, and retrieve one thing, and that should be enough to validate the functionality.  With all of these inserts, it makes it hard to see what we are actually validating.